test/e2e/steps/taskCreation.steps.js

```diff
@@ -0,0 +1,93 @@
+const BoardPage = require('../pages/BoardPage');
+const Context = require('../../data/Context');
+const DashboardPage = require('../pages/DashboardPage');
+const LoginPage = require('../pages/LoginPage');
+const SystemInteractions = require('../constants/SystemInteractions');
+const TaskAsserts = require('../asserts/Task.assert');
+const TaskPage = require('../pages/TaskPage');
+const Utils = require('../common/Utils');
+const { Given, When, Then } = require('cucumber');
+
+Given(/^user is in the main page$/, function () {
+  LoginPage.open(Context.page.endpoints.base);
+  browser.deleteCookies();
+});
+
+When(/^user creates "([^"]*)" list$/, function (listName) {
+  DashboardPage.addListButton.waitForExist(3000);
+  DashboardPage.addListButton.click();
+  DashboardPage.newListButton.click();
+  DashboardPage.setNewListInput(listName);
+  browser.keys(SystemInteractions.ENTER_KEY_PRESS);
+});
+
+When(/^user creates "([^"]*)" board$/, function (boardName) {
+    DashboardPage.setTaskName('      ');
+    DashboardPage.boardTab.click();
+    DashboardPage.addStatusButton.click();
+    BoardPage.setStatusNameInput(boardName);
+    browser.keys(SystemInteractions.ENTER_KEY_PRESS);
+    DashboardPage.modalText.click();
+    DashboardPage.confirmNewStatus.click();
```

Comment:
There are extra tabulations here

Reply:
Damn! you're right, I completely forgot that we are working with 2 spaces, great catch!!

Reply:
DONE